src/shared/components/custom-button/custom-button.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react'
+import { CustomButtonProps } from './custom-button.types'
+
+const CustomButton: FC<CustomButtonProps> = ({ value, onClick, type, styles = {}, classNames = '' }) => {
+    const handleClick = () => onClick?.()
+
+    return (
+        <button
+            style={{ ...styles }}
+            className={`px-3 py-2 border rounded-md text-lg ${type == 'secondary' ? 'bg-transparent border-gray-700 text-gray-700' : 'bg-gray-700 text-white'} ${classNames}`}
```

Comment:
one more requirement, need to add simple button with styles of text color black and bg transparent

Reply:
ok but it will be similar to the secondary button